xcdat/dataset.py

```diff
@@ -415,7 +415,7 @@ def _parse_xml_for_nc_glob(xml_path: Union[str, pathlib.Path]) -> str:
     Returns
     -------
     str
```

Comment:
I'm not sure how this is denoted, but I think it should be `str | List`

Reply:
Thanks for the checking! Yes, that is correct. In the last [commit](https://github.com/xCDAT/xcdat/pull/460/commits/37d6fbab67f3057f743cdacffd1a1e56166be96a#diff-e5b713e4bf50c61361496f77fdadcf9d3ffc2fcb39c4c953932cd780ae951e55R402) I revised this to `def _parse_xml_for_nc_glob(xml_path: Union[str, pathlib.Path]) -> Union[str, list]:` (`str | List` syntax works for python >= 3.10, but fails for python <= 3.9)